examples/public-repository-complete-example/main.tf

```diff
@@ -95,25 +134,35 @@ module "repository" {
   ]
 }
 
-
-
-resource "github_team" "team" {
-  name        = "private-repository-with-teams-test-team"
-  description = "This team is created with terraform to test the terraformn-github-repository module."
-  privacy     = "secret"
+# -----------------------------------------------------------------------------
+# TEST B
+# We are creating a repository using some defaults defined in locals
+# -----------------------------------------------------------------------------
+locals {
+  defaults = {
+    homepage_url       = "https://github.com/mineiros-io"
+    private            = false
+    allow_merge_commit = true
+    gitignore_template = "Terraform"
+    license_template   = "mit"
+    topics             = ["terraform", "integration-test"]
+  }
 }
 
-# ---------------------------------------------------------------------------------------------------------------------
-# TEAM MEMBERSHIP
-# We are adding two members to this team. terraform-test-user-1 and terraform-test-user-2 which are both existing users
-# and already members of the GitHub Organization terraform-test that is an Organization managed by Mineiros.io to run
-# integration tests with Terragrunt.
-# ---------------------------------------------------------------------------------------------------------------------
```

Comment:
not sure why this fails locally but not on CI.. as i am using the same token.. ? any ideas?

Reply:
okay... i know... the ci user has admin to both our orgs and i accidentally was using our main org.... :skull: ..we should create one for testing-only with limited access just top the test organization and one for our main org.. ;)